src/util/strencodings.h

```diff
@@ -50,8 +50,26 @@ std::vector<unsigned char> DecodeBase64(const char* p, bool* pf_invalid = nullpt
 std::string DecodeBase64(const std::string& str, bool* pf_invalid = nullptr);
 std::string EncodeBase64(Span<const unsigned char> input);
 std::string EncodeBase64(const std::string& str);
-std::vector<unsigned char> DecodeBase32(const char* p, bool* pf_invalid = nullptr);
-std::string DecodeBase32(const std::string& str, bool* pf_invalid = nullptr);
+
+/**
+ * Base32 decode.
+ * If `allow_nopad` is true, then the input is allowed to have length that is not
```

Comment:
This comment is outdated?

Reply:
Indeed! I renamed the comment so that the default behavior is with a `true` argument, to be in line with `EncodeBase32()`.